docs/integrations/saas-cloud/proofpoint-tap.md

```diff
@@ -161,9 +161,7 @@ To set up [Cloud to Cloud Integration Proofpoint TAP Source](/docs/send-data/hos
 
 ## Installing the Proofpoint TAP app
 
-This section provides instructions on how to install the Proofpoint TAP app, as well as examples of each of the dashboards. The app pre-configured searches and dashboards provide easy-to-access visual insights into your data.
-
-{@import ../../reuse/apps/app-install.md}
+{@import ../../reuse/apps/app-install-v2.md}
 
 ## Viewing Proofpoint TAP dashboards
 
```

Comment:
Can you delete this content and just add the import component?

Reply:
@shivani-sumo